src/lib.rs

```diff
@@ -41,6 +41,7 @@
 //! - `linux-static-hidraw`: uses statically linked `hidraw` backend on Linux (default)
 //! - `linux-shared-libusb`: uses dynamically linked `libusb` backend on Linux
 //! - `linux-shared-hidraw`: uses dynamically linked `hidraw` backend on Linux
+//! - `linux-shared-udev`: uses dynamically linked `udev` backend on Linux
```

Comment:
hm, udev is not really a backend for this, I`d rather call the backend *linux* and describe it as having no dependencies to the hidapi C library.